puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/rubygem-katello-nightly-release.groovy

```diff
@@ -0,0 +1,4 @@
+def owner_repo = 'Katello/katello'
```

Comment:
Would we be better off naming these files `rubygem-katello.groovy` since the data here is not specific to a nightly-release and could be re-used? Except I suppose maybe the branch.

Reply:
I'm still against the rubygem prefix because that's a RH-thing. This will be weird on packages that are also built on Debian. IMHO we should be using the source repo name rather than the result. Perhaps we should also be using the GH org but that's unlikely to be giving name conflicts. The branch could be a good thing though if we do start releasing "nightly" versions for released branches (1.18/3.7).

Reply:
I gave it the same name as this job to denote it's currently only related to the `rubygem-katello-nightly-release` job. But I do agree that if we decide to reuse it in the future, it could be renamed to something more generic.

Reply:
Last question, would it be cleaner then to just put these into the yaml itself?

Reply:
Eh, I retract that question as I don't think I like it after mulling.